bundle/manifests/opentelemetry-operator.clusterserviceversion.yaml

```diff
@@ -21,7 +21,7 @@ metadata:
     containerImage: quay.io/opentelemetry/opentelemetry-operator
     createdAt: "2020-12-16T13:37:00+00:00"
     description: Provides the OpenTelemetry components, including the Collector
-    operators.operatorframework.io/builder: operator-sdk-v1.10.0+git
+    operators.operatorframework.io/builder: operator-sdk-v1.9.0+git
```

Comment:
@templarfelix looks like you are using an older version of `operator-sdk,` make build on your local system downgraded the operator-sdk version. :/ 
Either manually change it to `1.10.0` or upgrade operator-sdk locally. :)

Reply:
changed